### PR TITLE
feature/CCE-492: updating build paths, structures, and starter page r…

### PR DESCRIPTION
**Pull request by HybrisRawls**
…efences to so that there are no 404s

### CODE REVIEW

File: docs/template/index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <link rel="icon" type="image/png" href="https://techne.yaas.io/images/favicons/favicon-16x16.png" sizes="16x16">
   <meta name="msapplication-TileColor" content="#da532c">
   <meta name="msapplication-TileImage" content="https://techne.yaas.io/images/favicons/mstile-144x144.png">
-  
+
   <title>Techne: Yaas Styleguide</title>
```
Comment (saadmhybris) [inline docs/template/index.html]: should we change this to  "Techné" for "SAP Hybris  Techné" while at it?

Comment (HybrisRawls) [inline docs/template/index.html]: we should change that here to. I'll update it now

File: dist/demo/starter-pages/cards.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <head>
-    <title>Techne: Layout Template - Cards</title>
+    <title>SAP Hybris Techné: Layout Template - Cards</title>
     <meta name="viewport" content="width=device-width">
-    <link rel="stylesheet" href="css/techne.min.css">
+    <link rel="stylesheet" href="techne/css/techne.min.css">
```
Comment (saadmhybris) [inline dist/demo/starter-pages/cards.html]: CSS path is still not correct. it should be "css/techne.min.css"

File: dist/demo/starter-pages/details-1.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <head>
-    <title>Techne: Layout Template - Details</title>
+    <title>SAP Hybris Techné: Layout Template - Details</title>
     <meta name="viewport" content="width=device-width">
-    <link rel="stylesheet" href="css/techne.min.css">
+    <link rel="stylesheet" href="techne/css/techne.min.css">
```
Comment (saadmhybris) [inline dist/demo/starter-pages/details-1.html]: CSS path is still not correct. it should be "css/techne.min.css"

File: dist/demo/starter-pages/details-2.html [hunk 4]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <head>
-    <title>Techne: Layout Template - Details</title>
+    <title>SAP Hybris Techné: Layout Template - Details</title>
     <meta name="viewport" content="width=device-width">
-    <link rel="stylesheet" href="css/techne.min.css">
+    <link rel="stylesheet" href="techne/css/techne.min.css">
```
Comment (saadmhybris) [inline dist/demo/starter-pages/details-2.html]: CSS path is still not correct. it should be "css/techne.min.css"

File: dist/demo/starter-pages/table.html [hunk 5]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <head>
-    <title>Techne: Layout Template - Table</title>
+    <title>SAP Hybris Techné: Layout Template - Table</title>
     <meta name="viewport" content="width=device-width">
-    <link rel="stylesheet" href="../public/dist/css/techne.min.css">
+    <link rel="stylesheet" href="techne/css/techne.min.css">
```
Comment (saadmhybris) [inline dist/demo/starter-pages/table.html]: CSS path is still not correct. it should be "css/techne.min.css"

Comment (HybrisRawls) [inline dist/demo/starter-pages/table.html]: this is correct. It was changed to match the new distribution paths

File: dist/demo/starter-pages/details-2.html (hunk 4 above)
Comment (HybrisRawls) [inline dist/demo/starter-pages/details-2.html]: this is correct. It was changed to match the new distribution paths

File: dist/demo/starter-pages/details-1.html (hunk 3 above)
Comment (HybrisRawls) [inline dist/demo/starter-pages/details-1.html]: this is correct. It was changed to match the new distribution paths

File: dist/demo/starter-pages/cards.html (hunk 2 above)
Comment (HybrisRawls) [inline dist/demo/starter-pages/cards.html]: this is correct. It was changed to match the new distribution paths

**Pull request merged**